services/ui-src/src/components/menus/Menu.tsx

```diff
@@ -0,0 +1,107 @@
+import { Link as RouterLink } from "react-router-dom";
+import {
+  Box,
+  Button,
+  Image,
+  Link,
+  Menu as MenuRoot,
+  MenuButton,
+  MenuItem,
+  MenuList,
+} from "@chakra-ui/react";
+import { MenuOption } from "components";
+import { useBreakpoint } from "utils";
+import accountCircleIcon from "assets/icons/icon_account_circle.png";
+import chevronDownIcon from "assets/icons/icon_arrow_down.png";
+import editIcon from "assets/icons/icon_edit_square.png";
+import logoutIcon from "assets/icons/icon_arrow_right_square.png";
+
+export const Menu = ({ handleLogout }: Props) => {
+  const { isMobile } = useBreakpoint();
+  return (
+    <MenuRoot offset={[8, 20]}>
+      <Box role="group">
+        <MenuButton
+          as={Button}
+          rightIcon={
+            <Image src={chevronDownIcon} alt="Arrow down" sx={sx.menuIcon} />
+          }
+          sx={sx.menuButton}
+          aria-label="my account"
+          data-testid="header-menu-dropdown-button"
```

Comment:
_If_ we can get the tests going without querying for Test IDs, we should strip them out of the markup as well.

```suggestion
```

Reply:
We do use `data-testid` a lot. I wonder why that became a common pattern.